src/drivers/rc/crsf_rc/Kconfig

```diff
@@ -3,3 +3,9 @@ menuconfig DRIVERS_RC_CRSF_RC
 	default n
 	---help---
 		Enable support for crsf rc
+
+config DRIVERS_RC_CRSF_RC_INJECT
```

Comment:
Maybe the partial match of DRIVERS_RC_CRSF_RC?

Reply:
> Maybe the partial match of DRIVERS_RC_CRSF_RC?

Yes, its breaks the naming convention as documented here
https://docs.px4.io/main/en/hardware/porting_guide_config#px4-kconfig-symbol-naming-convention